src/controllers/trainee/routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import Trainee from './Controller';
```

Comment:
import this as trainee (class instance).

Reply:
Done required changes.